### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Edgup**
In the methods.py, I fixed the edit label functionality by adding an empty case for deletion and fixing the select folder function to correctly work within the right model. By doing that, I had to make a few changes in the builder.py, classification.py, and datasets.py to allow the build function to work correctly too within the right model.

### CODE REVIEW

File: GUI/builder.py [hunk 1]
```diff
@@ -83,6 +84,8 @@ def builder(folder, NGRAMS, GAMMA, BATCH_SIZE, LEARNING_RATE, EMBED_DIM, EPOCHS,
      	
     train_dataset, test_dataset = _setup_datasets(data=data_file, root=root, ngrams=NGRAMS, vocab=None)
 
+    print(len(train_dataset))
```
Comment (jtanderson) [inline GUI/builder.py]: Looks like generic debug statement. Supposed to still be here?

File: GUI/builder.py [hunk 2]
```diff
@@ -158,6 +160,6 @@ def builder(folder, NGRAMS, GAMMA, BATCH_SIZE, LEARNING_RATE, EMBED_DIM, EPOCHS,
     Statistics.test_cat_val = test_comp
 
     # Saves the model
-    torch.save(model, root + "model")
-    torch.save(train_dataset.get_vocab(), root+"vocab")
-    return Statistics
\ No newline at end of file
+    torch.save(model, "model")
```
Comment (jtanderson) [inline GUI/builder.py:162]: Are we confident that the full path won't be needed here? Anyway way for the "model" variable to be corrupted/changed in a way that's not intended?

File: GUI/datasets.py [hunk 3]
```diff
@@ -70,8 +70,8 @@ def get_vocab(self):
         return self._vocab
 
 def _setup_datasets(data, root, ngrams=1, vocab=None, include_unk=False, rebuild=False):
-	train_csv_path = root + 'train.csv'
-	test_csv_path = root + 'test.csv'
+	train_csv_path = 'train.csv'
```
Comment (jtanderson) [inline GUI/datasets.py:73]: Similar comments as above: is there a case where the user could change where the model is supposed to be loaded from?

File: GUI/methods.py [hunk 4]
```diff
@@ -206,7 +206,7 @@ def addLabel(self):
 			fd.write(newLabel)
 			fd.close()
 		else:
-			fd.open('./.data/' + self.CLASS_NAME + '/labels.txt', 'a+')
+			fd = open('labels.txt', 'a+')
```
Comment (jtanderson) [inline GUI/methods.py:209]: Similar comment: this assumes we want the file adjacent to this file?

**Review by jtanderson — CHANGES_REQUESTED** [review]

**Comment by jtanderson** [comment]
@syntux Pinging you to give a second look and thoughts.

File: GUI/builder.py (hunk 1 above)
Comment (Edgup) [reply GUI/builder.py]: I forgot about that, I'll remove it.

File: GUI/methods.py (hunk 4 above)
Comment (syntux) [reply GUI/methods.py:209]: Should be . method instead of = ?

File: GUI/builder.py (hunk 2 above)
Comment (Edgup) [reply GUI/builder.py:162]: The path is correct since the current directory has been changed to the specified model using os.chdir() in the select folder function of methods.py. I made sure to test the current directory throughout various points using print. As of right now, the model variable shouldn't be corrupted since it's in the models directory

File: GUI/methods.py [hunk 5]
```diff
@@ -233,12 +236,12 @@ def delLabel(self):
 		if self.CLASS_NAME == '':
 			fd = open('labels.txt', 'w')
 		else:
-			fd = open('./.data/' + self.CLASS_NAME + '/labels.txt', 'a+')
+			fd = open('labels.txt', 'w')
```
Comment (syntux) [inline GUI/methods.py:239]: Use append ("a+") again or is 'w' for write fine?

File: GUI/methods.py (hunk 4 above)
Comment (syntux) [reply GUI/methods.py:209]: Nvm this should probably be =

File: GUI/builder.py (hunk 2 above)
Comment (jtanderson) [reply GUI/builder.py:162]: Great!

**Review by jtanderson — APPROVED** [review]
